### ISSUE TITLE
deperate this plugin in favor of `eslint-plugin-svelte` with correct parser

### ISSUE THREAD

**Issue by JounQin** [sveltejs/eslint-plugin-svelte3#184]
https://github.com/ota-meshi/eslint-plugin-svelte
https://github.com/ota-meshi/svelte-eslint-parser

**Comment by Jojoshua** [comment]
Agreed. I just noticed this plugin throwing errors causing linting to fail for a while. I just switched and it's working now plus it caught a lot of problems that should have fallen under our standard rules.  This plugin doesn't seem necessary anymore.

**Comment by multiplehats** [comment]
Can vote for this too; `ota-meshi/eslint-plugin-svelte` worked absolutely terrific.

**Comment by aradalvand** [comment]
I second this. This plugin also fails for things like indentation (see the issue I created #188), whereas on the other hand, `eslint-plugin-svelte` seems to work flawlessly.

**Comment by DetachHead** [comment]
@JounQin there's a typo in the issue title that makes this issue difficult to find. i couldn't find it by searching "deprecate"

**Comment by shinebayar-g** [comment]
Hmm didn't know there was another version. Sveltekit is installing https://github.com/sveltejs/eslint-plugin-svelte3 by default. I see the mentioned repositories are already redirecting to https://github.com/sveltejs/eslint-plugin-svelte, https://github.com/sveltejs/svelte-eslint-parser I guess those are official repositories now?

**Comment by ota-meshi** [comment]
`eslint-plugin-svelte` is now Svelte official ESLint plugin 🎉 
https://github.com/sveltejs/eslint-plugin-svelte
